tests/components/persistent_notification/test_notify.py

```diff
@@ -0,0 +1,28 @@
+"""The tests for the notify.persistent_notification platform."""
+from homeassistant.components import notify
+import homeassistant.components.persistent_notification as pn
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+
+async def test_async_send_message(hass: HomeAssistant):
+    """Test sending a message to persistent notify platform."""
+    await async_setup_component(hass, pn.DOMAIN, {"core": {}})
+    await async_setup_component(hass, notify.DOMAIN, {})
+    await hass.async_block_till_done()
+
+    message = {"message": "Hello", "title": "Test notification"}
+    await hass.services.async_call(notify.DOMAIN, pn.NOTIFY_SERVICE_NAME, message)
+    await hass.async_block_till_done()
+
+    notifications = hass.data[pn.DOMAIN]["notifications"]
```

Comment:
Get the state of the notification in the state machine `hass.states` instead. Don't access integration details in the test.